src/commands/deploy.ts

```diff
@@ -25,6 +28,17 @@ export default class Deploy extends Command {
     'frontend-refs-path': flags.string({
       default: './frontend/src/refs.terrain.json',
     }),
+    // "no-sync": flags.boolean({
+    //   description: "do not sync the contracts to the frontend.",
+    //   default: false,
+    // }),
+    sync: flags.boolean({
+      description: 'sync the contracts to the frontend.',
+      default: false,
+    }),
```

Comment:
```suggestion
    sync: flags.boolean({
      description: 'sync the contracts to the frontend.',
      default: true,
    }),
```

Also think this should default to true